src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj

```diff
@@ -35,6 +35,13 @@
           StandardOutputImportance="Low">
       <Output TaskParameter="ConsoleOutput" PropertyName="DnneCompilerCommand" />
     </Exec>
+
+    <Exec Command="bash -c 'source &quot;$(RepositoryEngineeringDir)/common/native/init-compiler.sh&quot; &quot;$(RepositoryEngineeringDir)/common/native&quot; $(TargetArchitecture) $(NativeCompiler) &amp;&amp; echo $LDFLAGS' 2>/dev/null"
```

Comment:
The other approach is that we echo "$CC;$LDFLAGS" on line 32 in one shot and use `ItemName` instead of `PropertyName`. But then accessing first and last item of item group is no fun.

cc @jkoritzinsky

Reply:
This looks fine to me. I'm fine with invoking the `init-compiler` script an extra time here to pull the info.